src/braket/ir/jaqcd/instructions.py

```diff
@@ -899,3 +900,47 @@ class Type(str, Enum):
         kraus = "kraus"
 
     type = Type.kraus
+
+
+class StartVerbatimBlock(CompilerDirective):
+    """
+    StartVerbatimBLock is a compiler instruction to start the code block which
+    will preserve the instruction within StartPreserveBlock and EndVerbatimBlock
+    from being modified in any way by the compiler.
+
+    Attributes:
+        type (str): The instruction type. default = "start_verbatim_block". (type) is optional.
+            This should be unique among all instruction types.
+
+    Examples:
+        >>> StartVerbatimBlock()
+    """
+
+    class Type(str, Enum):
+        start_verbatim_block = "start_verbatim_block"
+
+    type = Type.start_verbatim_block
+
+    directive: str = "StartVerbatimBlock"
+
+
+class EndVerbatimBlock(CompilerDirective):
+    """
+    EndVerbatimBlock is a compiler instruction to mark the end of the code block
+    which will preserve the instruction within StartPreserveBlock and EndVerbatimBlock
+    from being  modified in any way by the compiler.
+
+    Attributes:
+        type (str): The instruction type. default = "end_verbatim_block". (type) is optional.
+            This should be unique among all instruction types.
+
+    Examples:
+        >>> EndVerbatimBlock()
+    """
+
+    class Type(str, Enum):
+        end_verbatim_block = "end_verbatim_block"
+
+    type = Type.end_verbatim_block
+
+    directive: str = "EndVerbatimBlock"
```

Comment:
Interestingly, you can use this to just do a copy-paste: `directive: str = __qualname__`